src/sentry/south_migrations/0514_backfill_attachment_group_id.py

```diff
@@ -0,0 +1,1534 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models, transaction
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        EventAttachment = orm['sentry.EventAttachment']
+        Event = orm['sentry.Event']
+
+        queryset = EventAttachment.objects.all()
```

Comment:
Should you exclude EventAttachments that have a group_id? That would help make this migration idempotent.

Reply:
Good idea, thanks